We do not need to build wheels during testing so let's remove tox-wheel.

--- twisted-24.3.0/tox.ini.orig
+++ twisted-24.3.0/tox.ini
@@ -19,8 +19,7 @@
 minversion=3.24.1
 requires=
     virtualenv>=20.7.2
-    tox-wheel>=0.6.0
-    tox < 4
+    tox
 skip_missing_interpreters=True
 envlist=lint, mypy,
     apidocs, narrativedocs, newsfragment,
@@ -44,7 +43,7 @@
 ;; dependencies managed by extras in pyproject.toml
 extras =
     ; The "nodeps" build still depends on PyHamcrest.
-    nodeps: test
+    test
 
     ; FIXME:https://github.com/twisted/twisted/issues/11583
     ; Once we no longer have custom `towncrier` check the dev_release deps
@@ -121,7 +120,7 @@
     posix: python -c "print('Running on POSIX (no special dependencies)')"
 
     ; Run tests without wrapping them using coverage.
-    nocov: python -m twisted.trial --temp-directory={envtmpdir}/_trial_temp --reporter={env:TRIAL_REPORTER:verbose} {env:TRIAL_ARGS:-j8} {posargs:twisted}
+    python -m twisted.trial --temp-directory={envtmpdir}/_trial_temp --reporter={env:TRIAL_REPORTER:verbose} {env:TRIAL_ARGS:-j8} {posargs:twisted}
 
     ; Run the tests wrapped using coverage.
     withcov: python {toxinidir}/admin/_copy.py {toxinidir}/admin/zz_coverage.pth {envsitepackagesdir}/zz_coverage.pth
